index.js

```diff
@@ -424,6 +435,12 @@ alexa.app = function(name) {
           if (typeof eventHandlerObject != "undefined" && typeof eventHandlerObject["function"] == "function") {
             return Promise.resolve(eventHandlerObject["function"](request, response));
           }
+        } else if (request.isPlaybackController()) {
+          var playbackControllerEvent = requestType.slice(19);
```

Comment:
I agree, but the linter was [complaining](https://travis-ci.org/alexa-js/alexa-app/jobs/208017686) about it (no-redeclare).

Given how much of this is duplicated from the audioPlayer, some additional refactor could probably combine these two in to some shared infrastructure.